plugin/src/automation/lift/liftrules.ml

```diff
@@ -207,11 +254,7 @@ let is_pack c env sigma trm =
          Util.on_snd Option.has_some (right_type trm)
        else
          sigma, false
-    | SwapConstruct _ ->
-       (* no packing *)
-       sigma, false
-    | CurryRecord ->
-       (* taken care of by constructor rule *)
+    | _ ->
```

Comment:
Right so this is weird. Why do we take care of packing in constructor rules for everything else but for algebraic ornaments we take care of it here? Think about this, either now or when we work on the formalization.

Reply:
Also looks like this rule will currently trigger for fwd for UnpackSigma; see if it does, and if so what it does

Reply:
I think we basically always want this to be in the constr rule instead. But maybe that is not feasible for rel indices? Maybe this exists just to deal with rel indices? If so we can handle it elsewhere?